### PR TITLE
Table with default foreign key naming strategy

### PR DESCRIPTION
**Pull request by dominicbartl**
The same error still occurs. Because of the quotes around "Post_id" postgres dosn't find the column post_id

``` bash
 1) bookshelf-cascade-delete with PostgreSQL client should not delete model and its dependents if an error is thrown on destroy:

      AssertionError: expected 'delete from "Category" where "Post_id" IN (select "id" from "Post" where "authorId" IN (\'3\')) - column "Post_id" does not exist' to be 'foobar'
      + expected - actual

      -delete from "Category" where "Post_id" IN (select "id" from "Post" where "authorId" IN ('3')) - column "Post_id" does not exist
      +foobar
```

### CODE REVIEW

**Comment by ricardogama** [comment]
This is not sync with the current tree of #7. Can you please rebase and only leave your commits to be merged?

File: test/index.js [hunk 1]
```diff
@@ -333,5 +332,13 @@ describe('bookshelf-cascade-delete', () => {
 
       sinon.restore(Model);
     });
+
+    it('should foreignKey', async () => {
```
Comment (ricardogama) [inline test/index.js:336]: What are you trying to accomplish here?

Comment (dominicbartl) [inline test/index.js:336]: This test is useless. I used it to check if forgeinKey is undfined.

**Comment by ricardogama** [comment]
@Bartinger I already spotted the bug, working on it. Thanks!

**Comment by ricardogama** [comment]
@Bartinger This bug is something different than what was fixed in #7, so I'll close this PR for now and open an issue to keep track of it.

**Comment by ricardogama** [comment]
@Bartinger Can you take a look at the comment I just made on #9? It seems that the schema you created was malformed.

**Pull request closed without merging**